src/python/aqueduct_executor/operators/connectors/data/execute.py

```diff
@@ -272,20 +275,27 @@ def run_load(
     )
     if len(inputs) == 0:
         raise Exception("Expected at least one input artifact!")
-    if len(inputs) > 2:
-        raise Exception("Unexpected number of inputs to save operator: %v.", len(inputs))
 
-    # Handle any parameterization of the save queries here. Currently, we only support
-    # the parameterization of the `table_name` for SQL connectors.
+    # Handle any parameterization of the save queries here.
     if len(inputs) > 1:
-        if not isinstance(spec.parameters, RelationalParams):
-            raise Exception("Only relational database resources support parameterized saves.")
-
-        assert (
-            len(spec.parameters.table) == 0
-        ), "A parameterized relational save spec should have an empty table name."
-        assert isinstance(inputs[0], str), "Relational saves can only have string parameters."
-        spec.parameters.table = inputs[0]
+        for idx, input in enumerate(inputs[:-1]):
+            if not isinstance(input, str):
+                raise Exception(
+                    "Unexpected input type %s to save at %s-th index." % (type(input), idx)
+                )
+
+        param_input_vals = [str(input) for input in inputs[:-1]]
+        if isinstance(spec.parameters, RelationalParams):
+            assert (
+                len(spec.parameters.table) == 0
+            ), "A parameterized relational save spec should have an empty table name."
+            spec.parameters.table = param_input_vals[0]
```

Comment:
Just confirming we are not supporting parametrizing a portion of the table name here

Reply:
correct